src/gSP.cpp

```diff
@@ -2288,8 +2288,8 @@ void _loadBGImage(const uObjScaleBg * _bgInfo, bool _loadScale)
 		gSP.bgImage.width = *REG.VI_WIDTH;
 		gSP.bgImage.height = (imageH * imageW) / gSP.bgImage.width;
 	} else {
-		gSP.bgImage.width = imageW - imageW%2;
-		gSP.bgImage.height = imageH - imageH%2;
+		gSP.bgImage.width = imageW & 0xFFFFFFFE;
+		gSP.bgImage.height = imageH - 1;
```

Comment:
I suppose that imageH - imageH%2 was done for purpose.
`imageH & 0xFFFFFFFE` would cause no questions.
I'm afraid that `imageH - 1` will cause regressions. I need to test it first.

Reply:
@gonetz 
There can be regressions but i have seen none. I have tested beetle adventure racing, sm64, mario tennis, smash bros, paper mario, yoshis story, mischief makers, bomberman and many other games, re2, doom, mario kart and goldeneye.